### PR TITLE
:bug: Fix typo in e2e environment config

### PR DESCRIPTION
**Pull request by Rozzii**

### CODE REVIEW

**Review by kashifest — APPROVED** [review]

File: scripts/environment.sh [hunk 1]
```diff
@@ -85,9 +85,9 @@ export PROVISIONING_URL_HOST="172.22.0.1"
 export CLUSTER_PROVISIONING_IP="172.22.0.2"
 export CLUSTER_URL_HOST="$CLUSTER_PROVISIONING_IP"
 
-export BARE_METAL_RPOVISIONER_URL_HOST="172.22.0.1"
+export BARE_METAL_PROVISIONER_URL_HOST="172.22.0.1"
```
Comment (kashifest) [inline scripts/environment.sh:88]: where are we using these? Shall we not change it there as well?

**Review by lentzi90 — COMMENTED** [review]
/lgtm

File: scripts/environment.sh (hunk 1 above)
Comment (Rozzii) [reply scripts/environment.sh:88]: These are used in dev-env and I have a PR that implements these names.

**Pull request merged**